versions/fix.nest.ver

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# Fix file subfolder versions
+source "${HOMEgfs:-}/versions/fix.ver"
+
+export am_ver=global-nest.20240419
+export orog_ver=global-nest.20240419
+export ugwd_ver=global-nest.20240419
```

Comment:
The vast majority of these files are not grid-specific. They're climatology files on their own grid. Do we really need our own copy of everything?

Reply:
This eliminates the need of "W384", "W768" etc. I think this simplifies the changes and make it easier to be accepted by EMC.
At the time I was working on this, I don't know what files are your new fix files. 

We can either (1) make a new copy of every climatology file as the files size is not that big and acceptable;
or (2) create a new version but make those climatology files as links.
We can further discuss this in a tag up. Thanks!

Reply:
I see no other way to get rid of the W grids, so I agree we'll have to have our own fix files unless EMC comes up with a better option.